index.js

```diff
@@ -1,33 +1,26 @@
-// Without "./", Node looks
-// in its global module folder
-const http = require("http");
-function handleRequestAndResponse(req, res) {
-    console.log(req.url);
-    const timestamp = new Date();
-    console.log("Received a Request");
-    res.end(`<h1>Hello World ${timestamp}</h1>`);  // .end() sends the response.
-                                          // It "tapes up the box and ships it".
+const nerds = require(`nerds`);
+const http = require(`http`);
+
+function getRandomInt(max) {
+    return Math.floor(Math.random() * Math.floor(max));
 }
-// What's a server??!?
-// A server is a piece of code
-// or a computer running a piece of code
-// that does something useful for YOU.
-// Usually, servers store
-// data and often manipulate that data.
-// const server = http.createServer(handleRequestAndResponse);
 const server = http.createServer((req, res) => {
-    console.log(req.url);
-    const timestamp = new Date();
-    console.log("I totally received a Request!!!!!!!!!!!!!!!!!!!");
-    res.end(`<h1>Hello World ${timestamp}</h1>`);  // .end() sends the response.
-                                          // It "tapes up the box and ships it".
+    res.end(`Your new Pokemon name is ${createName()}`)
 });
 
-// http.createServer() accepts a function and returns a "server"
-// When the server receives a Request:
-// 1. It puts together *most* of a Response.
-// 2. It calls *your* function and passes it the Request and Response
+function createName() {
+    const pokeName = nerds.resolve('Pokemon', 1).include(['name']).asArray()[0].name;
+    const starName = nerds.resolve('Star Wars', 1).include(['name']).asArray()[0].name;
+    const starNameSplit = starName.split(' ');
+    if (starNameSplit.length > 1) {
```

Comment:
Nice way to handle SW chars with both first and last names. Consider adding a comment to make that clear?